.ghal.rules.json

```diff
@@ -18,6 +18,12 @@
           "(?i).*master\/aspnetcore\/host-and-deploy\/blazor.*": {
             "labels-add": "Blazor"
           },
+          "(?i).*master\/aspnetcore\/mvc\/views\/tag-helpers\/built-in\/component.md": {
```

Comment:
Let's hold off on the TH entry until the topic goes up. The other entry you added looks good.